content/tracing/setup/docker.md

```diff
@@ -46,35 +46,85 @@ As with DogStatsD, traces can be submitted to the agent from other containers ei
 ### Using Docker links
 
 ```
-docker run  --name my_container           \
-            --all_your_flags              \
-            --link dd-agent:dd-agent    \
+docker run  --name my_container \
+            --all_your_flags \
+            --link dd-agent:dd-agent \
             my_image
 ```
 
-It expose `DD_AGENT_PORT_8126_TCP_ADDR` and `DD_AGENT_PORT_8126_TCP_PORT` as environment variables. Your application tracer can be configured to submit to this address.
+It expose `DD_AGENT_PORT_8126_TCP_ADDR` and `DD_AGENT_PORT_8126_TCP_PORT` as environment variables. Your application tracer must be configured to submit traces to this address.
 
-An example in Python:
+
+#### Python Configuration
```

Comment:
@l0k0ms probably, should I remove `Configuration` everywhere?

Reply:
yup, it's not really usefull

Reply:
done!